.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs
```

Comment:
The syntax for ignoring the `logs` directory is incorrect. It should be `logs/` instead of `/logs`. This ensures that log files are not pushed to GitHub.